include/core/CConcurrentQueue.h

```diff
@@ -160,6 +159,9 @@ class CConcurrentQueue final : private CNonCopyable {
     static bool always(const T&) { return true; }
 
 private:
+    std::size_t m_QueueCapacity;
```

Comment:
How about moving documentation from @tparam here, e.g. `fixed queue capacity` and same for `m_NotifyCapacity`.